Rakefile

```diff
@@ -81,3 +83,46 @@ end
 
 desc 'By default, gather dependencies, build and test'
 task :default => [:deps, :test, :install]
+
+task :sync_consul_deps do
+  consul_vendor_contents = JSON.parse(File.read(File.join(ENV["GOPATH"], "src", "github.com", "hashicorp", "consul", "vendor", "vendor.json")))
+
+  p2_repo_dir = File.join(ENV["GOPATH"], "src", "github.com", "square", "p2")
+  p2_godep_contents = JSON.parse(File.read(File.join(p2_repo_dir, "Godeps", "Godeps.json")))
+
+  # build a map of package names to revisions from consul's vendor.json
+  consul_deps = {}
+  consul_vendor_contents["package"].each do |package|
+    consul_deps[package["path"]] = package["revision"]
+  end
+
+  # now iterate over our Godeps.json and find any versions that mismatch and print them
+  p2_godep_contents["Deps"].each do |package|
+    package_name = package["ImportPath"]
+    consul_rev = consul_deps[package_name]
```

Comment:
What behavior will this script have when a new dependency is encountered in Consul?